connectors/src/api/slack_channels_linked_with_agent.ts

```diff
@@ -131,11 +131,11 @@ const _patchSlackChannelsLinkedWithAgentHandler = async (
         req,
         res,
         {
+          status_code: 400,
```

Comment:
wouldn't that be a 500 here? not being able to join a channel for variable possible reasons?